src/main/java/com/google/sps/servlets/FamilyServlet.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for creating new tasks. */
+@WebServlet("/family")
+public class FamilyServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        UserService userService = UserServiceFactory.getUserService();
+        
+        String familyName = request.getParameter("family-name");
+        String creatorEmail = userService.getCurrentUser().getEmail();
+        long timestamp = System.currentTimeMillis();
+
+        ArrayList<String> memberEmails = new ArrayList(Arrays.asList(creatorEmail)); 
+
+        Entity familyEntity = new Entity("Family");
+        familyEntity.setProperty("name", familyName);
+        familyEntity.setProperty("memberEmails", memberEmails);
```

Comment:
At some point, I think you will want to abstract operations on family entity out into its own class (and probably have a class that makes it easier to access things on family, even if it's just a wrapper around Family with getters and setters.

You can use a DAO pattern: https://www.tutorialspoint.com/design_pattern/data_access_object_pattern.htm